src/main/java/fi/dy/masa/tweakeroo/tweaks/MiscTweaks.java

```diff
@@ -222,6 +228,28 @@ private static void doPotionWarnings(PlayerEntity player)
         }
     }
 
+    public static boolean isEntityAllowedByAttackingRestriction(EntityType<?> type)
+    {
+        if(MiscTweaks.ENTITY_TYPE_ATTACK_RESTRICTION.isAllowed(type) == false)
```

Comment:
I always use a space in the if/for/while before the opening parenthesis. (Same thing in the Mixin.)